### ISSUE TITLE
Detected incorrect number of misses

### ISSUE THREAD

**Issue by ThereGoesMySanity** [firedigger/osuReplayAnalyzer#15]
I ran Cookiezi's HDHR choke on Blue Zenith through the program and it detected 4 misses as opposed to 2. It looks to me like the CS equation might not be totally correct.

EDIT: After doing some testing of my own, I don't think it's the CS equation.

**Comment by firedigger** [comment]
Yeah, honestly, when I was checking myself it looked like float error, but I am not sure myself really.

**Comment by ThereGoesMySanity** [comment]
Changing the shift value to 0 gets rid of all of the false misses.

EDIT: the highest I could get it to without false misses was 1.5

**Comment by firedigger** [comment]
What do you mean by "shift"?

**Comment by ThereGoesMySanity** [comment]
the offset for stacks - it's called shiftValue in Beatmap.cs

**Comment by firedigger** [comment]
Stack shift is there for a reason though - in the game stacked notes have slight different coordinates. I think that my stack calculation might be incorrect, causing this, will figure it out eventually.
